### PR TITLE
libmd, bump version, add libVersion(Compat), defineDebugInfoPackage

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Builds ok on x86_gcc2 and x86 (secondary arch), tests run ok on both arch's

File: app-crypt/libmd/libmd-1.0.0.recipe [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	cmd:make
+	cmd:sed
+	"
+
+defineDebugInfoPackage libmd$secondaryArchSuffix \
+	"$libDir"/libmd.so.$libVersion
+
+BUILD()
+{
+	autoreconf -vfi
```
Comment (alaviss) [inline app-crypt/libmd/libmd-1.0.0.recipe]: Is this still necessary?

File: app-crypt/libmd/libmd-1.0.0.recipe [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+systems either on their libc libraries and lacking on others like GNU systems.
+
+The currently provided message digest algorithms are:
+- MD2
+- MD4
+- MD5
+- RIPEMD-160
+- SHA-1
+- SHA-2 (SHA-256, SHA-384 and SHA-512)
+"
```
Comment (alaviss) [inline app-crypt/libmd/libmd-1.0.0.recipe]:
```diff
-- SHA-2 (SHA-256, SHA-384 and SHA-512)
-"
+- SHA-2 (SHA-256, SHA-384 and SHA-512)"
```

File: app-crypt/libmd/libmd-1.0.0.recipe [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+HOMEPAGE="https://www.hadrons.org/software/libmd/"
+COPYRIGHT="2009, 2011, 2018 Guillem Jover
+	2000-2001 Aaron D. Gifford
+	2001 Markus Friedl
+	2001 The NetBSD Foundation
+	1997, 2003, 2004 Todd C. Miller
+	Poul-Henning Kamp
+	Colin Plumb
+	Steve Reid
+	"
```
Comment (alaviss) [inline app-crypt/libmd/libmd-1.0.0.recipe]:
```diff
-	Steve Reid
-	"
+	Steve Reid"
```

File: app-crypt/libmd/patches/libmd-1.0.0.patchset [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+diff --git a/src/sha2.c b/src/sha2.c
+index 029d0d4..5cbbe2b 100644
+--- a/src/sha2.c
++++ b/src/sha2.c
+@@ -36,6 +36,10 @@
+ 
+ #include <sys/types.h>
+ 
++#ifdef __HAIKU__
++#include <posix/endian.h>
```
[the remark this answers is not in the document]
Comment (Begasus) [reply app-crypt/libmd/patches/libmd-1.0.0.patchset]: Will fix, remainder of previous recipe ...

File: app-crypt/libmd/libmd-1.0.0.recipe [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+LICENSE="BSD (3-clause)
+	BSD (2-clause)
+	ISC
+	Public Domain"
```
[the remark this answers is not in the document]
Comment (Begasus) [reply app-crypt/libmd/libmd-1.0.0.recipe]: I saw the Beerware license mentioned in there :) Should they be added separately into "licenses"?

File: app-crypt/libmd/libmd-1.0.0.recipe (hunk 1 above)
Comment (Begasus) [reply app-crypt/libmd/libmd-1.0.0.recipe]: Nope :)

**Pull request merged**